cargo-dist/src/main.rs

```diff
@@ -230,7 +249,7 @@ fn cmd_generate_ci(cli: &Cli, _args: &GenerateCiArgs) -> Result<(), miette::Repo
         installers: cli.installer.iter().map(|ins| ins.to_lib()).collect(),
         announcement_tag: cli.tag.clone(),
     };
-    let args = cargo_dist::GenerateCiArgs {};
+    let args = cargo_dist::GenerateCiArgs { check: args.check };
     do_generate_ci(&config, &args)
 }
```

Comment:
Similarly it would be nice if this was calling `do_generate(&[GenerateStyle::Ci], args.check)` so that the alias boils away immediately.